composables/use-likecoin-session-api.ts

```diff
@@ -275,6 +275,25 @@ export function useLikeCoinSessionAPI() {
     })
   }
 
+  async function sendCollectorMessage({
+    classId,
+    paymentId,
+    message,
+    wallet,
+    claimToken,
+  }: {
+    classId: string
+    paymentId: string
+    message: string
+    wallet: string
+    claimToken: string
+  }) {
+    return fetch.value<string>(`/likernft/book/purchase/class/${classId}/memo/${paymentId}?token=${claimToken}`, {
```

Comment:
```suggestion
    return fetch.value<string>(`/likernft/book/purchase/class/${classId}/message/${paymentId}?token=${claimToken}`, {
```

Changed to `/message` in https://github.com/likecoin/likecoin-api-public/pull/1007

Reply:
[🎨 Update sendCollectorMessage endpoint URL](https://github.com/likecoin/3ook-com/pull/262/commits/262f87bc70fb179a10c57ac6a19d79351a1d4259)